src/cljs/rems/navbar.cljs

```diff
@@ -98,6 +100,7 @@
      [skip-navigation]
      [:div.navbar-top-bar [:div.navbar-top-left] [:div.navbar-top-right]]
      [:div.navbar-wrapper.container-fluid
+      [:div "This is a test"]
```

Comment:
Typically we do not commit this kinds of tests. I would drop them out from the PR. We don't squash PRs when we merge them.